frontend/mocks/handlers.ts

```diff
@@ -0,0 +1,23 @@
+import { rest } from 'msw'
+
+const REST_SERVER = process.env.REST_SERVER || ''
+
+export const handlers = [
+  rest.post(`${REST_SERVER}/api/login`, async (req, res, ctx) => {
```

Comment:
I'm guessing this is being used by the LoginForm test? 🤔 

I kinda get why you went this route. But what if instead of mocking the whole rest server, you just mock it in a per request basis?

meaning in LoginForm, you mock the success API response `in` the actual LoginForm test. Instead of having a global server mock 🤔